src/main/java/jenkins/branch/BranchSource.java

```diff
@@ -78,7 +77,12 @@ public SCMSource getSource() {
      * @return the strategy.
      */
     public BranchPropertyStrategy getStrategy() {
-        return strategy;
+        return strategy != null ? strategy : new DefaultBranchPropertyStrategy(new BranchProperty[0]);
```

Comment:
Could. The number of instances in the VM is probably only a handful so I saw no need to bother.